README.md

```diff
@@ -121,8 +121,8 @@ The General flow is described in the diagram:
 The changes to the driver are almost purely changing the search strategy it uses to find the extension .dll file.
 It is very reliant on nuget to place things correctly, so it's worth calling out the specific contracts:
 
-1. dotnet driver expects that a restore of an extensions package will have the unpacked package directory (`NUGET_PACKAGE_CACHE/packages/dotnet-extension-package/1.0.0`) copied to the dotnet-extensions package hive (`NUGET_PACKAGE_CACHE/dotnet-extensions/dotnet-extension-package/1.0.0`)
-2. dotnet driver expects that a project.lock.json for the `dotnet-extension-package` will exist in the dotnet-extensions package hive (`NUGET_PACKAGE_CACHE/dotnet-extensions/dotnet-extension-package/1.0.0/project.lock.json`)
+1. dotnet driver expects that a restore of an extensions package will have the unpacked package directory (`~/.nuget/packages/dotnet-extension-package/1.0.0`) copied to the dotnet-extensions package hive (`~/.nuget/dotnet-extensions/dotnet-extension-package/1.0.0`)
```

Comment:
I was using NUGET_PACKAGE_CACHE as a strawman for this location. Thanks for the update.

`~` seems ambiguous though. Does it refer to $HOME on unix systems? What of accounts which do not have a $HOME variable or even a HOME folder at all (`useradd` doesn't create one by default).

My intention is to call out the ambiguity, I'm not aware if there's already a solution here or it's strictly unsupported :smile: